nomos-services/cryptarchia-consensus/src/relays.rs

```diff
@@ -108,32 +108,34 @@ impl<
     >
 where
     BlendAdapter: blend::BlendAdapter<Network: BlendNetworkAdapter>,
-    BS: BlobSelect<BlobId = DaPool::Item> + Clone,
+    BS: BlobSelect<BlobId = DaPool::Item>,
     ClPool: RecoverableMempool<BlockId = HeaderId>,
     ClPool::RecoveryState: Serialize + for<'de> Deserialize<'de>,
-    ClPool::BlockId: Debug,
     ClPool::Item: Debug + DeserializeOwned + Eq + Hash + Clone + Send + Sync + 'static,
-    ClPool::Key: Debug + 'static,
+    ClPool::Key: 'static,
     ClPool::Settings: Clone,
     ClPoolAdapter: MempoolAdapter<Payload = ClPool::Item, Key = ClPool::Key>,
     DaPool: RecoverableMempool<BlockId = HeaderId>,
-    DaPool::RecoveryState: Serialize + for<'de> Deserialize<'de>,
     DaPool::BlockId: Debug,
     DaPool::Item: Debug + DeserializeOwned + Eq + Hash + Clone + Send + Sync + 'static,
     DaPool::Key: Debug + 'static,
+    //
```

Comment:
leftover?

Reply:
Yup, from the rebas, still cleaning up stuff.